server/channels/app/user.go

```diff
@@ -1445,7 +1445,11 @@ func (a *App) UpdatePassword(rctx request.CTX, user *model.User, newPassword str
 		return model.NewAppError("UpdatePassword", "api.user.update_password.failed.app_error", nil, "", http.StatusInternalServerError)
 	}
 
-	hashedPassword := model.HashPassword(newPassword)
+	hashedPassword, err := model.HashPassword(newPassword)
+	if err != nil {
+		// can't be password length (checked in IsPasswordValid)
```

Comment:
Can you point me to where we validate the password for a new user? (i.e. signing up)

Reply:
https://github.com/mattermost/mattermost/blob/master/server/channels/app/users/users.go#L63